element_deeplabcut/model.py

```diff
@@ -683,28 +693,41 @@ def make(self, key):
         # ID model and directories
         dlc_model = (Model & key).fetch1()
 
-        task_mode, analyze_video_params, output_dir = (PoseEstimationTask & key).fetch1(
-            "task_mode", "pose_estimation_params", "pose_estimation_output_dir"
+        assert dlc_model["project_path"], (
+            "Your model table must have a 'project_path'"
+            + "field pointing to a DLC directory"

```

Comment:
Why is this necessary if the [table field](https://github.com/ttngu207/element-deeplabcut/blob/main/element_deeplabcut/model.py#L332) is not nullable? Should we instead be asserting here that the path exists?

Reply:
You're right, we don't need this

Reply:
fixed